src/extensions/registries/status-bar-registry.ts

```diff
@@ -3,7 +3,18 @@
 import React from "react";
 import { BaseRegistry } from "./base-registry";
 
-export interface StatusBarRegistration {
+export interface StatusBarComponents {
+  Item?: React.ComponentType;
+}
+
+export interface StatusBarRegistrationV2 {
```

Comment:
Does this _have_ to be exported? When it comes time to actually remove the `item` field, won't we just eliminate this interface and redefine `StatusBarRegistration` accordingly? If so it would be better if we didn't export something we'll eventually remove.